src/Umbraco.ModelsBuilder/Umbraco/PublishedModelUtility.cs

```diff
@@ -40,7 +41,15 @@ public static PublishedContentType GetModelContentType(PublishedItemType itemTyp
             }
         }
 
-        public static PublishedPropertyType GetModelPropertyType<TModel, TValue>(PublishedContentType contentType, Expression<Func<TModel, TValue>> selector)
+        [Obsolete("Use the overload taking IPublishedContentType instead")]
```

Comment:
Do we need to obsolete anything here? This is brand new code/APIs that has never been shipped so we shouldn't really need to ship an obsoleted method.

Reply:
You're right.. We don't..